pkgs/top-level/all-packages.nix

```
@@ -10851,6 +10851,10 @@ with pkgs;
     SDL = SDL_sixel;
   };
 
+  witness = callPackage ../tools/security/witness {
+    buildGoModule = buildGo117Module;
+  };
```

`1.17` has recently been made the default.
```suggestion
  witness = callPackage ../tools/security/witness { };
```